### PR TITLE
german translation

### PR DESCRIPTION
**Pull request by uDEV2019**

### CODE REVIEW

File: lib/l10n/app_de.arb [hunk 1]
```diff
@@ -0,0 +1,24 @@
...
+  "copy": "In die Zwischenablage kopieren",
+  "search": "Suchen...",
+  "history": "Verlauf",
+  "deleteAll": "Alles löschen",
+  "deleteTitle": "Möchten Sie den Verlauf wirklich löschen?",
+  "deleteDescription": "Sie können diese Aktion nicht rückgängig machen.",
+  "clear": "Verlauf leeren",
+  "cancel": "Abbrechen",
+  "error": "Etwas ist schief gelaufen!",
+  "copyToast": "In die Zwischenablage kopiert.",
```
Comment (MahanRahmati) [inline lib/l10n/app_de.arb:23]: "copyToast": "In die Zwischenablage kopiert."

**Comment by MahanRahmati** [comment]
Hello there!
Thanks for contributing! 
Please make the requested changes.

File: lib/l10n/app_de.arb (hunk 1 above)
Comment (uDEV2019) [reply lib/l10n/app_de.arb]: fixed

**Comment by MahanRahmati** [comment]
LGTM

**Pull request merged**